bin/verify-exercise-example

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env sh
```

Comment:
How do you feel about upgrading to `bash`?

Reply:
No feelings at all :D